### ISSUE TITLE
Default Skills get muddled when skill names are changed

### ISSUE THREAD

**Issue by 123mind123** [schultzcole/FVTT-Skill-Customization-5e#14]
In a game I am running I changed Charisma to Creativity and Cha to Cre. This has resulted in the game defaulting things like Deception to Str and despite my players changing it each game it keeps flipping it back.

**Comment by schultzcole** [comment]
Hmm, I'll have to investigate this. How did you go about changing the ability? Did you just edit the language file or did you use a world script or module?

**Comment by 123mind123** [comment]
Yeah I changed the language file. "DND5E.AbilityCha" has remained the same while : "Charisma" is now  : "Creativity"
 
 edit: and ': "cha"' became ': "cre"'
